packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -547,8 +547,15 @@ def create_http_request(builder: RequestBuilderType) -> List[str]:
         return retval
 
     def serialize_headers(self, builder: RequestBuilderType) -> List[str]:
-        retval = ["# Construct headers"]
+        retval = []
```

Comment:
Don't set content-type when its value is "multipart/form-data" since `requests` will do it by default. Otherwise `requests` will fail.